repos/rust-lang/compiler-team.toml

```diff
@@ -0,0 +1,13 @@
+org = "rust-lang"
+name = "compiler-team"
+description = "Contains a static site with procedures, policies, working groups, planning documents and minutes of the compiler team."
+bots = []
+
+[access.teams]
+compiler = "write"
```

Comment:
please give compiler contributors access to the repo

Reply:
Done!